content/en/account_management/billing/custom_metrics.md

```diff
@@ -5,68 +5,199 @@ aliases:
+
+The number of custom metrics from [COUNT][1], [RATE][2], and [GAUGE][3] metric types is calculated with the same logic.
+
+The number of unique tag value combinations submitted for a GAUGE metric with this tagging scheme is **four**:
+
+* `host:A`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:200`
+* `host:B`, `endpoint:X`, `status:400`
+* `host:B`, `endpoint:Y`, `status:200`
+
+This results in `request.Latency` reporting **four distinct custom metrics**.
+
+### Effect of adding tags
+
+Adding tags **may not** result in more custom metrics. Your count of custom metrics usually scales with the most granular or detailed tag. Suppose you are measuring temperature in the US, and you have tagged your temperature metric by country and region. You submit the following to Datadog:
+
+| Metric Name   | Tag Values                         |
+|---------------|------------------------------------|
+| `temperature` | `country:USA`, `region: Northeast` |
+| `temperature` | `country:USA`, `region: Southeast` |
+
+Suppose you wanted to add the tag `City` which has three values: `NYC`, `Miami`, and `Orlando`. Adding this tag increases the number of custom metrics as it provides more detail and granularity to your dataset as shown below:
+
+| Metric Name   | Tag Values                                          |
+|---------------|-----------------------------------------------------|
+| `temperature` | `country:USA`, `region: Northeast`, `city: NYC`     |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Orlando` |
+| `temperature` | `country:USA`, `region: Southeast`, `city: Miami`   |
+
+The count of custom metrics reporting from `temperature` scales with the most granular tag, `city`.
+
+Now suppose you also wanted to tag your temperature metric by `state` (which has two values: `NY` and `Florida`). You’re now tagging temperature by the set of tags: `country`, `region`, `state`, and `city`. Adding the state tag doesn’t increase the level of granularity already present in your dataset provided by the city tag.
+
+To obtain the temperature in Florida, you can simply recombine the custom metrics of:
+
+* `temperature:{country:USA, state:Florida, city:Orlando}`
+* `temperature{country:USA, state:Florida, city:Miami}`
+
+**Note**: Reordering tag values doesn’t add uniqueness. The following combinations are the same custom metric:
+
+*  `temperature{country:USA, state:Florida, city:Miami}`
+*  `temperature{state:Florida, city:Miami, country:USA}`
+
+[1]: /developers/metrics/types/?tab=count#metric-submission-types
+[2]: /developers/metrics/types/?tab=rate#metric-submission-types
+[3]: /developers/metrics/types/?tab=gauge#metric-submission-types
+{{% /tab %}}
+{{% tab "Histogram" %}}
+
+**A HISTOGRAM metric generates by default five custom metrics** for each unique combination of metric name and tag values to support the Agent-side aggregations `max`, `median`, `avg`, `95pc`, and `count`. [Learn more about HISTOGRAM metric type][1].
```

Comment:
I'm a bit hesitant to have only **A HISTOGRAM metric generates by default five custom metrics**  bolded in case people stop reading the sentence at that point. 

**A HISTOGRAM metric generates by default five custom metrics for each unique combination of metric name and tag values** to support the Agent-side aggregations `max`, `median`, `avg`, `95pc`, and `count`. [Learn more about HISTOGRAM metric type][1].